eng/common/TestResources/New-TestResources.ps1

```diff
@@ -611,8 +614,17 @@ try {
         }
     }
 
+    if ($UseUserCredentials) {
+        if ($TestApplicationId) {
+            Write-Warning "The specified TestApplicationId '$TestApplicationId' will be ignored when UseUserCredentials is set."
```

Comment:
```suggestion
            Write-Warning "The specified TestApplicationId '$TestApplicationId' will be ignored when '$UseUserCredentials' is set."
```

Be consistent.

Reply:
But won't that evaluate the value of the parameter? I'm trying to print the parameter name in the second instance.

Reply:
I would say `the -UseUserCredentials parameter is set` since it matches how it's set.

Reply:
we don't use that style in any other logs though.

Reply:
Use a backtick:
```powershell
"foo `$NotAVariable` bar"
```